### ISSUE TITLE
Make style guide more prominent

### ISSUE THREAD

**Issue by thewavelength** [capnproto/capnproto#1106]
Reading the [KJ style guide](https://github.com/capnproto/capnproto/blob/master/style-guide.md) has been a life safer. Would have been nice to read it on day one using this library instead of day five as it would have saved probably ten hours of wrong design for my application. Especially the part of just reading the most common header comments as it clarifies a lot of things which are important.

Maybe it is worth linking this document more prominently in the documentation.

Thanks for making this library!

**Comment by kentonv** [comment]
Agreed, in fact, I'm hoping soon to create a whole web site for KJ, incorporating the style guide and a bunch of other documentation.
